with open("exercise3ExceptionHandlingModulesFileManagementandPythonModulesStandardLibraries/p12/test_file.txt","r") as file:
    for i in file:
        print(i)